src/Web/Telegram/API/Bot/API.hs

```diff
@@ -68,182 +73,179 @@ newtype Token = Token Text
 telegramBaseUrl :: BaseUrl
 telegramBaseUrl = BaseUrl Https "api.telegram.org" 443 ""
 
--- | Type for token
-type TelegramToken = Capture ":token" Token
-
 -- | Telegram Bot API
 type TelegramBotAPI =
```

Comment:
If you don't want to trait token as part of entry point = base url, then you have to invent a wrapper containing token on top of Servant.ClientM, so user can write a client without passing a token each line.